chime/chime-connection.c

```diff
@@ -1069,6 +1070,16 @@ chime_connection_send_message_async(ChimeConnection *self,
 					   CHIME_IS_ROOM(obj) ? "room" : "conversation",
 					   chime_object_get_id(obj));
 	JsonNode *node = json_builder_get_root(jb);
+	JsonObject *json_obj = json_node_get_object(node);
+	if (additional_json) {
+		GList *members = json_object_get_members(additional_json);
+		while (members != NULL) {
+			gchar* member_name = (gchar*)members->data;
+			JsonNode *node_2 = json_object_dup_member(additional_json, member_name);
```

Comment:
I don't like node_2 as a variable name...

Reply:
Me neither, temporary code that (almost) became permanent :P
Renamed